both/_round.js

```diff
@@ -0,0 +1,175 @@
+
+// Add functions to the Mongo object, using transform (see http://docs.meteor.com/#/full/mongo_collection)
+Round = function(doc) {
+  _.extend(this, doc);
+};
+_.extend(Round.prototype, {
+
+  format: function() {
+    return wca.formatByCode[this.formatCode];
+  },
+  properties: function() {
+    return wca.roundByCode[this.roundCode];
+  },
+  eventName: function() {
+    return this.eventCode ? wca.eventByCode[this.eventCode].name : '';
+  },
+  eventSolveTimeFields: function() {
+    return this.eventCode ? wca.eventByCode[this.eventCode].solveTimeFields : undefined;
+  },
+  isUnstarted: function() {
+    return this.status === wca.roundStatuses.unstarted;
+  },
+  isOpen: function() {
+    return this.status === wca.roundStatuses.open;
+  },
+  isClosed: function() {
+    return this.status === wca.roundStatuses.closed;
+  },
+});
+
+
+
+MIN_ROUND_DURATION_MINUTES = 30;
```

Comment:
Could and should. I would expose MIN_ROUND_DURATION_MINUTES &
DEFAULT_ROUND_DURATION_MINUTES, and inline the unused DEFAULT_ROUND_NTHDAY.